main/ui/src/main/resources/fxml/preferences_general.fxml

```diff
@@ -40,8 +40,8 @@
 		</HBox>
 
 		<HBox spacing="6" alignment="CENTER_LEFT">
-			<Label text="%preferences.general.pwBackend"/>
-			<ChoiceBox fx:id="pwBackendChoiceBox"/>
+			<Label text="%preferences.general.keychainBackend"/>
+			<ChoiceBox fx:id="keychainBackendChoiceBox"/>
```

Comment:
Not sure if this works (my IDE marks it red and I haven't tested it):

```suggestion
			<ChoiceBox fx:id="keychainBackendChoiceBox"  disable="${keychainBackendChoiceBox.items.size &lt;= 1}"/>
```

Reply:
No, this is not possible. One can not access the size from the FXML.
I would suggest using a BooleanBinding as I did [here](https://github.com/BeyerMart/cryptomator/commit/947354bcc696353cbc2605d1daea7d216e1a58ea).
But then you have watch out for the NullPointerException at `keychainBackendChoiceBox.getItems().size() > 1`